apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
@tanyaparker - might be misunderstanding your comment but you do need to hit both.  "Lock stage" just puts everything in the "locked" column.  You'd still need to hit "save" for your changes to apply.